From 75f4f703ebf46815d0287736384d3ba11e3f4162 Mon Sep 17 00:00:00 2001
From: Brad Fitzpatrick <bradfitz@golang.org>
Date: Fri, 28 Oct 2016 03:56:30 +0000
Subject: [PATCH 31/38] [release-branch.go1.7] doc: remove mention of Go 1.6.3
 working on Sierra

We thought it would at the time, but then Beta 4 changed the ABI
again, so it wasn't true in practice.

Fixes #17643

Change-Id: I36b747bd69a56adc7291fa30d6bffdf67ab8741b
Reviewed-on: https://go-review.googlesource.com/32238
Reviewed-by: Russ Cox <rsc@golang.org>
Reviewed-on: https://go-review.googlesource.com/32270
---
 doc/go1.7.html | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/doc/go1.7.html b/doc/go1.7.html
index 762c1ec..6839c5e 100644
--- a/doc/go1.7.html
+++ b/doc/go1.7.html
@@ -67,8 +67,7 @@ This change has no effect on the correctness of existing programs.
 
 <p>
 Go 1.7 adds support for macOS 10.12 Sierra.
-This support was backported to Go 1.6.3.
-Binaries built with versions of Go before 1.6.3 will not work
+Binaries built with versions of Go before 1.7 will not work
 correctly on Sierra.
 </p>
 
-- 
2.7.4

